src/ducks/account.ts

```diff
@@ -26,7 +26,7 @@ export const fetchAccountAction = createAsyncThunk<
       stellarAccount = await dataProvider.fetchAccountDetails();
     } catch (error) {
       return rejectWithValue({
-        errorMessage: error.response?.detail || error.toString(),
+        errorString: error.response?.detail || error.toString(),
```

Comment:
Oh, this is the first time I'm noticing this. Horizon error responses are a lot more detailed, and require more processing, than just this. On Vibrant we use a helper function to process them:

https://github.com/stellar/mobile-wallet/blob/master/src/helpers/getErrorString.ts